waveform_editor/gui/plotter_edit.py

```diff
@@ -1,16 +1,28 @@
+from io import StringIO
+
+import holoviews as hv
+import panel as pn
 import param
+from holoviews import streams
+from panel.viewable import Viewer
+from ruamel.yaml import YAML
 
-from waveform_editor.gui.plotter_base import PlotterBase
+from waveform_editor.tendencies.piecewise import PiecewiseLinearTendency
 from waveform_editor.waveform import Waveform
 
 
-class PlotterEdit(PlotterBase):
+class PlotterEdit(Viewer):
```

Comment:
Drag and drop had quite some issues. As discussed, I've tried to fix them in #69